src/cli/mod.rs

```diff
@@ -4,4 +4,5 @@ mod fmt;
 mod tc;
 
 pub use error::CLIError;
+pub use fmt::Fmt;
```

Comment:
we do use it outside `cli` module? if not, we don't need the `pub` identifier (we can use `pub(crate)` instead)

Reply:
Solved. Thanks.